packages/bierzo-wallet/src/config/staging.json

```diff
@@ -31,7 +31,29 @@
         "chainId": "ethereum-eip155-4",
         "name": "Rinkeby",
         "node": "https://rinkeby.infura.io/v3/143ecb3d2d864557b4b140f731530909",
-        "scraper": "https://api-rinkeby.etherscan.io/api"
+        "scraper": "https://api-rinkeby.etherscan.io/api",
+        "ethereumOptions": {
+          "erc20s": [
+            {
+              "contractAddress": "0x0c8184c21a51cdb7df9e5dc415a6a54b3a39c991",
+              "decimals": 18,
+              "symbol": "AVO",
+              "name": "Avocado"
+            },
+            {
+              "contractAddress": "0xc778417e063141139fce010982780140aa0cd5ab",
+              "decimals": 18,
+              "symbol": "WETH",
+              "name": "Wrapped Ether"
+            },
+            {
+              "contractAddress": "0x1f9061B953bBa0E36BF50F21876132DcF276fC6e",
+              "decimals": 0,
+              "symbol": "ZEENUS",
+              "name": "Zeenus"
```

Comment:
No issue but just FYI: the emoji is part of the token name in the contract: https://rinkeby.etherscan.io/token/0x1f9061B953bBa0E36BF50F21876132DcF276fC6e#readContract

But we can rename the name to "Zeenus" without the emoji locally, no problem. Only the symbol is important to associate amounts with tokens.